src/Compilers/CSharp/Test/Semantic/Semantics/SemanticErrorTests.cs

```diff
@@ -2769,19 +2769,23 @@ namespace ns1
 ";
             // TODO (tomat): EOFUnexpected shouldn't be reported if we enable parsing global statements in namespaces
             DiagnosticsUtils.VerifyErrorsAndGetCompilationWithMscorlib(test,
```

Comment:
>VerifyErrorsAndGetCompilationWithMscorlib [](http://example.com/codeflow?start=29&length=41)

Please switch the test to base-line form used by the test below.

Reply:
Done